### PR TITLE
[WIP] Implement Official Content

### PR DESCRIPTION
**Pull request by anity99**
* **Addressed Issue(s)**: [#1301](https://github.com/rathena/rathena/issues/1301), [#1313](https://github.com/rathena/rathena/issues/1313), [#1302](https://github.com/rathena/rathena/issues/1302)

* **Server Mode**: Renewal

* **Description of Pull Request**: 
I've finished some official content and added it in vAthena. Now I share it with the rAthena community, all script is Vietnamese, so I need to translate it to English, need a bit time to finish.

I don't make sure all scripts are matching with Aegis. But we can make it closest that we can.

**Heroes' Trails**
- [ ] Devil's Tower instance
- [x] Airship Assault instance
- [x] Sarah and Fenrir instance (Atemo PR)

**Decisive Battle**
- [ ] Dimensional Travel quest
- [ ] Episode 14.3: Decisive Battle
- [ ] Bios Island instance
- [ ] Morse Cave instance
- [ ] Temple of Demon God instance
- [ ] Nightmarish Jitterbug instance

**Banquet for Heroes**
- [ ] Episode 16.1: Banquet for Heroes
- [ ] Room of Consciousness instance
- [ ] Sky Fortress instance 

**Unclassified**
- [ ] Infinite Space instance
- [ ] Poring Village instance
- [ ] Remnant of Azit instance

**Other**
- [x] Update database includes: item, mob, instance, quest, mercenary
- [ ] Convert database to rAthena's styles
- [ ] Recheck & clean up

**Rock Ridge**
- Are planned ^^.

**Credits**
- idAthena: database infomation

### CODE REVIEW

File: db/const.txt [hunk 1]
```diff
@@ -2313,3 +2313,6 @@ ARCWANDCLAN	2
 GOLDENMACECLAN	3
 CROSSBOWCLAN	4
 JUMPINGCLAN	5
+
+IG_PrizeOfHero	453
+IG_Squad_Prize	456
```
Comment (Lemongrass3110) [inline db/const.txt]: Please move those over to https://github.com/rathena/rathena/blob/master/src/map/itemdb.h#L728 and export them via  https://github.com/rathena/rathena/blob/master/src/map/script_constants.h#L3814

Thank you!

**Comment by Lemongrass3110** [comment]
CI is failing because of missing database entries:
https://ci.appveyor.com/project/rAthenaAPI/rathena/build/master-1802-qwlxnfmt/job/aw1acduvxt7kp1n1#L830

File: db/const.txt (hunk 1 above)
Comment (anity99) [reply db/const.txt]: Okay, I didn't update document for a long time 😁

**Comment by exneval** [comment]
@anity99 for mob_db.txt please update/convert it to use the new rA monster mode, and also Mob ATK1 and ATK2 need to update using rA style, you can use divinepride as references

**Comment by anity99** [comment]
Thank you for hint, I'm working on it

**Comment by Atemo** [comment]
The scripts are customs ? from aegis?

File: db/re/instance_db.txt [hunk 2]
```diff
@@ -23,3 +23,11 @@
 16,Horror Toy Factory,3600,300,1@xm_d,111,22
 17,Faceworm's Nest,3600,300,1@face,112,374
 18,Ghost Palace,3600,300,1@spa,42,196
+19,Bios Island,7200,300,1@dth1,19,94,1@dth2,1@dth3
```
Comment (Atemo) [inline db/re/instance_db.txt]: From my data Bios Island (or Isle of Bios) instance duration is 3600s, entrance at 17,93

File: db/re/instance_db.txt [hunk 3]
```diff
@@ -23,3 +23,11 @@
...
+21,Nightmarish Jitterbug,3300,300,1@jtb,16,18
```
Comment (Atemo) [inline db/re/instance_db.txt]: Instance duration 3600s

**Comment by Atemo** [comment]
hmm why did you close your PR? It was interesting

**Comment by Mikegyver** [comment]
yup. this PR is interesting... especially for the instance part..

**Comment by Mikegyver** [comment]
@anity99 what is your id name on rA forum? i would love to invite u to join a discord group which quite related on discussion some of the official implementation & leaks..

**Comment by anity99** [comment]
I'm back

**Comment by Atemo** [comment]
Personally I would prefer to split the scripts in several PR (let's say one instance by PR) to be able to review them and merge them on master faster.

Actually we have some scripts waiting in our archive.
- Airship Assault instance (by Secret)
- Devil's Tower instance
- Dimensional Travel quest
- Some ep14.3 npcs converted from aegis
- Bios Island instance
- Episode 16.1: Banquet for Heroes
- Room of Consciousness instance (actually this instance is used as daily quest and-  for me - is a part of ep16.1 quests)
- Rockridge npcs

But you're welcome to share yours scripts to us so we can discuss it. We will use yours if they are better than ours of course.

On side note we don't share the scripts above yet because:
- we want to follow ragnarok episode
- we don't have enough feedback in the scripts already shared to be able to merge them and work on something else (maybe no people is testing them?)

**Comment by anity99** [comment]
Okay, I'll create PR for per script. But with the script that you had, I'll not share it again, because all of its are in Vietnamese, I need to convert it to English, it takes a lot of time, I don't want the waste of time for the duplicate work.

But I'll focus testing all your PR for master faster.

**Comment by kisuka** [comment]
@Atemo these are more than likely custom as leaks aren't be distributed currently. Customs are fine though as long as they match official as close as possible. I encourage those contributing to resort to walkthrough conversions going forward then use possible future leaks to correct things later.

**Comment by anity99** [comment]
According to Atemo, I will divide my PR into smaller parts (per PR). So I close my PR now. For further information, feel free to contact me at jenkijo#6479

**Pull request closed without merging**